docs/features/os.rst

```diff
@@ -1,14 +1,60 @@
 Integrating with the OS
 =======================
 
-- how to redirect output
-- executing OS commands from within ``cmd2``
-- editors
-- paging
-- exit codes
-- Automation and calling cmd2 from other CLI/CLU tools via commands at
-  invocation and quit
+How to redirect output
+----------------------
 
+See :ref:`features/redirection:Output Redirection and Pipes`
+
+Executing OS commands from within ``cmd2``
+------------------------------------------
+
+``cmd2`` includes a ``shell`` command which executes it's arguments in the
+operating system shell::
+
+    (Cmd) shell ls -al
+
+If you use the default :ref:`features/shortcuts_aliases_macros:Shortcuts`
+defined in ``cmd2`` you'll get a ``!`` shortcut for ``shell``, which allows you
+to type::
+
+    (Cmd) !ls -al
+
+
+Editors
+-------
+
+``cmd2`` includes the built-in ``edit`` command which uns a text editor and
```

Comment:
Change uns to runs

Reply:
I wish Sphinx had a built-in spell checker.  There was an add-on, but I think its out of date and unmaintained.